packages/account/src/Styles/account.scss

```diff
@@ -418,6 +424,42 @@ $MIN_HEIGHT_FLOATING: calc(
                 }
             }
         }
+        &__phone-verification-btn {
```

Comment:
Is possible, lets move these styles to respective files rather than adding ore styles to this css file